.gitpod.yml

```
@@ -1,11 +1,13 @@
 tasks:
   - init: |
-      mvn -am -pl war,bom -P quick-build clean install
+      mvn -B -T 1C -DskipTests package
     command: |
-      mvn -pl war jetty:run -Dhost=0.0.0.0
+      mvn -pl war jetty:run -Djetty.http.port=8080 -DskipTests
```

The jetty:run command does not benefit from `-DskipTests`.  I believe that the `-Dhost=0.0.0.0 is to allow my local web browser to connect with the Jenkins that is running in Gitpod.  Did you test this to confirm that it works?
```suggestion
      mvn -pl war jetty:run -Djetty.http.port=8080 -DskipTests
```